decidim-budgets/app/views/decidim/budgets/projects/index.html.erb

```diff
@@ -8,9 +8,9 @@
 <% content_for :aside do %>
   <h1 class="h3 decorator"><%= t("decidim.budgets.projects.projects_for", name: translated_attribute(budget.title)) %></h1>
 
-  <%= render layout: "decidim/shared/filters", locals: { filter_sections: , search_variable: :search_text_cont } do %>
-      <%= hidden_field_tag :order, order, id: nil, class: "order_filter" %>
-    <% end %>
+  <%= render layout: "decidim/shared/filters", locals: { filter_sections: , search_variable: :search_text_cont, skip_to_id: "projects" } do %>
+    <%= hidden_field_tag :order, order, id: nil, class: "order_filter" %>
```

Comment:
The projects filtering is happening using ajax. Changing the filters on projects, will not change the value of the hidden input. 
[Screencast from 2023-08-28 00-39-19.webm](https://github.com/decidim/decidim/assets/105683/d09c11d4-9c6a-4757-93d0-28dbf91307d4)